tests/test_posts.py

```
@@ -0,0 +1,211 @@
+        headers = {"Authorization": "Bearer invalid_token"}
+        response = client.get("/posts/me", headers=headers)
+        assert response.status_code == 401
+
+
+class TestGetSinglePost:
+    """GET /posts/{post_id} 테스트"""
+
+    def test_get_single_post_success(self, client, mock_posts_data):
+        """게시글 조회 성공"""
+        with patch("routers.posts.read_json", return_value=mock_posts_data):
+            with patch("routers.posts.write_json") as mock_write:
```

Ruff (0.14.13)

Local variable `mock_write` is assigned to but never used

Remove assignment to unused variable `mock_write`

(F841)